zephyr-make

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/make -rf
```

Comment:
This requires bash.

Putting effort into non-portable solutions is not forbidden, but it has
a poor return on investment compared to portable solutions.

EDIT: This is not a review comment, just a comment.

Reply:
I'm confused here...you are saying use of a she-bang comment at the beginning of a scrip is bash-specific?

Reply:
I think Sebastian means that it requires a UNIX derivative. Windows does not support she-bangs AFAIK.

Reply:
Just for the purpose of discussion (with the understanding this isn't merged into mainline):

> I think Sebastian means that it requires a UNIX derivative. Windows does not support she-bangs AFAIK.

I don't think this valuation is all-encompassing. This requires a POSIX system, POSIX being a standard for "portable operating systems". Portable in a sense that a user can switch from one such system to another, and still end up in a familiar environment, being able to follow familiar workflow. There's simply no other standard for such systems, but POSIX.

Window specifically supported POSIX-compatible subsystems since NT times (native Microsoft's subsystem, UWIN, Cygwin, I can mention your truly's http://pw32.sourceforge.net/ which I hacked when Cygwin was too slow on my P5-100, recently Ubuntu native subsystem).